examples/tfengine/generated/team/example-prod-secrets/main.tf

```diff
@@ -27,14 +29,32 @@ resource "random_password" "db" {
   special = true
 }
 
+# Create the project and optionally enable APIs, create the deletion lien and add to shared VPC.
+module "project" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 8.1.0"
+
+  name                    = "example-prod-secrets"
+  org_id                  = ""
```

Comment:
Is it supposed to have an empty `org_id`?

Reply:
Yes, folder_id is set so we don't need to set org, but the CFT module requires it (for some resources that we don't setup).